src/components/PDFView/index.js

```diff
@@ -114,12 +114,13 @@ class PDFView extends Component {
         // If we're requesting a password then we need to hide - but still render -
         // the PDF component.
         const pdfContainerStyle = this.state.shouldRequestPassword
-            ? [styles.PDFView, this.props.style, styles.invisible]
-            : [styles.PDFView, this.props.style];
+            ? [styles.PDFView, styles.noSelect, this.props.style, styles.invisible]
+            : [styles.PDFView, styles.noSelect, this.props.style];
```

Comment:
@Santhosh-Sellavel There should be no outline showing when the div is focused. I can't reproduce that outline and neither your first video test.